src/main/java/com/synopsys/integration/alert/channel/jira/JiraIssueHandler.java

```diff
@@ -90,12 +90,13 @@ public JiraIssueHandler(
     }
 
     public String createOrUpdateIssues(final FieldAccessor fieldAccessor, final MessageContentGroup content) throws IntegrationException {
-        final String projectName = fieldAccessor.getString(JiraDescriptor.KEY_JIRA_PROJECT_NAME).orElse("");
+        final String projectName = fieldAccessor.getString(JiraDescriptor.KEY_JIRA_PROJECT_NAME).orElse(StringUtils.EMPTY);
         final PageOfProjectsResponseModel projectsResponseModel = projectService.getProjectsByName(projectName);
         final ProjectComponent projectComponent = projectsResponseModel.getProjects()
                                                       .stream()
-                                                      .findFirst()
-                                                      .orElseThrow(() -> new AlertException(String.format("No projects matching '%s' were found", projectName)));
+                                                      .filter(project -> projectName.equals(project.getName()))
+                                                      .findAny()
+                                                      .orElseThrow(() -> new AlertException(String.format("No project named '%s' was found", projectName)));
```

Comment:
AlertException could be an AlertFieldException. That would make the error easier to understand in the UI.